src/components/tiles/geometry/geometry-content.tsx

```diff
@@ -454,7 +453,7 @@ export class GeometryContentComponent extends BaseComponent<IProps, IState> {
   };
 
   private handleTitleChange = (title?: string) => {
-    title && this.applyChange(() => this.getContent().updateTitle(this.state.board, title));
+    title && this.props.model.setTitle(title);
```

Comment:
Note: This is is simplified now. Changing the title is no longer done view a change in the content model which is monitored and then `TileModel#setTitle` is called by monitoring code.  You'll see that other code simplification below. Because of that, it isn't necessary to wrap the call in a `applyChange`.